### PR TITLE
parser/tutorialConclusion

### PR DESCRIPTION
**Pull request by MarcelDiessner**

### CODE REVIEW

**Review by EduardKrieger — DISMISSED** [review]

**Review by GuentherJulian — APPROVED** [review]

**Comment by GuentherJulian** [comment]
There are some conflicts that must be resolved.
And could you add the method for the console runner (just an empty method, https://github.com/devonfw-tutorials/tutorial-compiler/issues/204) and maybe the wiki runner (https://github.com/devonfw-tutorials/tutorial-compiler/issues/205), too

**Comment by MarcelDiessner** [comment]
> There are some conflicts that must be resolved.
> And could you add the method for the console runner (just an empty method, #204) and maybe the wiki runner (#205), too

I think the console runner version is not needed because i fill the finish.md file in the destroy method and not in a seperate one

**Review by GuentherJulian — DISMISSED** [review]

File: engine/parser.def [hunk 1]
```diff
@@ -61,6 +65,14 @@ stepline
 function "function call"
   = [^\r\n(]+[(][^\r\n)]*[)] { return text(); }
 
+conclusion
+	= "====" ___
+    conclusionlines
+  	"====" ___ 
+    
+conclusionlines
+  = descriptionline+ { return { "conclusionlines": text()}; }
```
Comment (SchettlerKoehler) [inline engine/parser.def:74]: I would duplicate the descriptionline token to avoid confusion.

File: engine/parser.def [hunk 2]
```diff
@@ -61,6 +65,14 @@ stepline
 function "function call"
   = [^\r\n(]+[(][^\r\n)]*[)] { return text(); }
 
+conclusion
+	= "====" ___
+    conclusionlines
+  	"====" ___
```
Comment (SchettlerKoehler) [inline engine/parser.def:71]: Use the blockmarker token

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

**Comment by SchettlerKoehler** [comment]
Please inform @ssarmokadam when this is finished and merged.

**Review by GuentherJulian — APPROVED** [review]

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**